bindings.properties is not included in release archive?

I was playing with testing the CentOS compatibility and noticed that if I download the release archive then `bindings.properties` is not in the release so `./bin/ycsb.sh` doesn't work.

I think #784 and #785 partially addressed this for the ycsb scripts themselves.

For reference here is what I was running:
```
docker run --rm -it centos:6 bash
yum -y install git java-1.8.0-openjdk-devel wget epel-release && wget http://repos.fedorapeople.org/repos/dchen/apache-maven/epel-apache-maven.repo -O /etc/yum.repos.d/epel-apache-maven.repo && yum -y install apache-maven
RELEASE=0.11.0; rm -rf /opt/ycsb-$RELEASE /opt/YCSB; wget -qO- https://github.com/brianfrankcooper/YCSB/releases/download/$RELEASE/ycsb-$RELEASE.tar.gz | tar xvfz - -C /opt && mv /opt/ycsb-$RELEASE /opt/YCSB; unset RELEASE
cd /opt/YCSB
./bin/ycsb.sh load basic -P workloads/workloada
```

The output from the `ycsb.sh` script is as follows:
```
# ./bin/ycsb.sh load basic -P workloads/workloada
grep: /opt/YCSB/bin/bindings.properties: No such file or directory
[ERROR] The specified binding 'basic' was not found.  Exiting.
```

--- a/binding-parent/datastore-specific-descriptor/src/main/resources/assemblies/datastore-specific-assembly.xml
+++ b/binding-parent/datastore-specific-descriptor/src/main/resources/assemblies/datastore-specific-assembly.xml
@@ -45,6 +45,14 @@ LICENSE file.
       </includes>
     </fileSet>
     <fileSet>
+      <directory>../bin</directory>
+      <outputDirectory>bin</outputDirectory>
+      <fileMode>0644</fileMode>
+      <includes>
+        <include>bindings.properties</include>
+      </includes>
+    </fileSet>
+    <fileSet>
       <directory>../workloads</directory>
       <outputDirectory>workloads</outputDirectory>
       <fileMode>0644</fileMode>

--- a/distribution/src/main/assembly/distribution.xml
+++ b/distribution/src/main/assembly/distribution.xml
@@ -42,6 +42,14 @@ LICENSE file.
       </includes>
     </fileSet>
     <fileSet>
+      <directory>../bin</directory>
+      <outputDirectory>bin</outputDirectory>
+      <fileMode>0644</fileMode>
+      <includes>
+        <include>bindings.properties</include>
+      </includes>
+    </fileSet>
+    <fileSet>
       <directory>../workloads</directory>
       <outputDirectory>workloads</outputDirectory>
       <fileMode>0644</fileMode>

